pkg/controllers/provisioning/provisioner.go

```diff
@@ -137,20 +136,13 @@ func (p *Provisioner) LaunchMachines(ctx context.Context, machines []*scheduler.
 	machineNames := make([]string, len(machines))
 	workqueue.ParallelizeUntil(ctx, len(machines), len(machines), func(i int) {
 		// create a new context to avoid a data race on the ctx variable
-		ctx := logging.WithLogger(ctx, logging.FromContext(ctx).With("provisioner", machines[i].Labels[v1alpha5.ProvisionerNameLabelKey]))
-		// register the provisioner on the context so we can pull it off for tagging purposes
-		// TODO: rethink this, maybe just pass the provisioner down instead of hiding it in the context?
```

Comment:
What was the resolution here?

Reply:
There's no reason to inject the provisioner into the context since now we can grab the provisioner from the `Machine` that is used in the `cloudprovider.Create()` call